### PR TITLE
add noetic to .travis.yml

### PR DESCRIPTION
**Pull request by k-okada**
Imported most of #5 functions, but it did not use `catkin_install_python` as commented on 
https://github.com/PR2/linux_networking/pull/5#discussion_r584112906

### CODE REVIEW

**Comment by v4hn** [comment]
I pushed #5 (with the correct package version attribute) on top of this to resolve the python3 errors.
Do you plan to merge the other noetic-CI prs before addressing issues or do you want to have them resolved in the same PR (to keep CI succeeding)?

File: .travis.yml [hunk 1]
```diff
@@ -5,21 +5,25 @@
...
-python:
-  - "3.4"
 env:
   global:
     - JOB_PATH=/tmp/devel_job
     - ABORT_ON_TEST_FAILURE=1
+    - INDEX_URL=https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml
   matrix:
+    - CHECK_PYTHON3_COMPILE=true
+    - ROS_DISTRO_NAME=indigo  OS_NAME=ubuntu OS_CODE_NAME=trusty ARCH=amd64 INDEX_URL=https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/6a93d17/index.yaml
```
Comment (v4hn) [inline .travis.yml:17]: Why bother to keep indigo in here at all?
I would propose to drop it altogether from CI (throughout the other repositories as well).

**Review by v4hn — APPROVED** [review]
Wow, you did a lot more work there than I expected, thanks!
Looks good and CI is green.

Please consider my inline comment before merging.

File: .travis.yml (hunk 1 above)
Comment (k-okada) [reply .travis.yml:17]: @v4hn we have indigo test, but it is actually failing, -> https://travis-ci.com/github/PR2/linux_networking/builds/218549093
This means CI goes green even if indigo fails, but if there is someone who still uses indigo, they could know what is wrong with it.

**Comment by v4hn** [comment]
> we have indigo test, but it is actually failing,

That's why I proposed to drop it. But I don't feel strong about it and your argument is right, the failing test might help someone.

**Pull request merged**